pool/pool_aio_test.go

```diff
@@ -266,7 +266,7 @@ func testPoolInterfaceWithAIO(t *testing.T, nodeAddr string) {
 		ctxTimeout, cancel := context.WithTimeout(context.Background(), defaultTimeOut)
 		defer cancel()
 
-		containerID = testCreateContainer(t, ctxTimeout, signer, cont, pool)
+		containerID = testCreateContainer(ctxTimeout, t, signer, cont, pool)
```

Comment:
tbh, i would not say that `ctx` is a preferable first arg to me in a test file. do not insist

Reply:
I agree, but linter doesn't

Reply:
yeah, that is why linters may [be ommited](https://github.com/nspcc-dev/neofs-node/blob/f751d711f28d242aead2f912f6019d91e36531b9/.golangci.yml#L9-L10) for a test file?

@roman-khimov, @cthulhu-rider

Reply:
I'd always use the same same of linters for tests. Just to not have some different styles/assumptions.